alb.tf

```diff
@@ -1,16 +1,19 @@
+// External LB for WAF
 resource "aws_alb" "default" {
-  name            = "${var.env}-${var.ecs_cluster_name}-alb"
+  count           = "${var.use_external_elb ? 1 : 0}"
+  name            = "${var.env}-${var.ecs_cluster_name}-external-alb"
   internal        = false
-  security_groups = ["${aws_security_group.alb_ecs_access.id}", "${join("", aws_security_group.alb_waf_access.*.id)}", "${join("", aws_security_group.alb_ons_access.*.id)}"]
+  security_groups = ["${aws_security_group.alb_ecs_access.id}", "${join("", aws_security_group.alb_ons_access.*.id)}"]
   subnets         = ["${var.public_subnet_ids}"]
 
   tags {
-    Name        = "${var.ecs_cluster_name}-alb"
+    Name        = "${var.ecs_cluster_name}-external-alb"
     Environment = "${var.env}"
   }
 }
 
 resource "aws_alb_listener" "default" {
+  count             = "${var.use_external_elb ? 1 : 0}"
```

Comment:
@bitdivision you need to also make this change here. 
https://github.com/ONSdigital/eq-terraform-ecs/pull/25/files#diff-6b34ad218f4c18c4778c00e2d11fe0a1R50

Reply:
Thanks!